experiment/builder_runner.py

```diff
@@ -935,15 +953,15 @@ def build_and_run_cloud(
         f'--real_project={project_name}',
     ]
 
-    if oss_fuzz_checkout.ENABLE_CACHING and (
-        oss_fuzz_checkout.is_image_cached(project_name, 'address') and
-        oss_fuzz_checkout.is_image_cached(project_name, 'coverage')):
-      logger.info('Using cached image for %s', project_name)
-      command.append('--use_cached_image')
+    # if oss_fuzz_checkout.ENABLE_CACHING and (
+    #     oss_fuzz_checkout.is_image_cached(project_name, 'address') and
+    #     oss_fuzz_checkout.is_image_cached(project_name, 'coverage')):
+    #   logger.info('Using cached image for %s', project_name)
+    #   command.append('--use_cached_image')
 
-      # Overwrite the Dockerfile to be caching friendly
-      oss_fuzz_checkout.rewrite_project_to_cached_project_chronos(
-          generated_project)
+    #   # Overwrite the Dockerfile to be caching friendly
+    #   oss_fuzz_checkout.rewrite_project_to_cached_project_chronos(
+    #       generated_project)
```

Comment:
could you leave a comment on why this is being commented out?

Reply:
Done, I will test if the cached images in the next PR.